lib/cfndsl/aws/types.yaml

```diff
@@ -1074,6 +1075,9 @@ Types:
  EC2Tag:
   Key: String
   Value: String
+ ElastiCacheTag:
```

Comment:
We definitely shouldn't be creating new named tags. AWS has taken to calling them generically resource tags now: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html
We use EC2Tag in most places. I vote for keeping that pattern for the moment.